src/STW.StubApi.IntegrationTests/HealthCheckTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace STW.StubApi.IntegrationTests;
+
+using System.Net;
+using System.Net.Http.Json;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Presentation.HealthChecks;
+
+[TestClass]
+public class HealthCheckTests
+{
+    private const string RequestUri = "/health";
+    private readonly HttpClient _httpClient;
+
+    public HealthCheckTests()
+    {
+        var factory = new StubApiWebApplicationFactory();
+        _httpClient = factory.CreateClient();
+    }
+
+    [TestMethod]
+    public async Task HealthCheck_ReturnsOkStatusWithCorrectBody_WhenCalled()
+    {
+        // Arrange / Act
```

Comment:
Would you not just specify Act here because there is no setup, just calling the system under test?